pkg/compute/rust.go

```diff
@@ -164,12 +164,27 @@ func (r Rust) Build(out io.Writer) error {
 	// Specify the toolchain using the `cargo +<version>` syntax.
 	toolchain := fmt.Sprintf("+%s", RustToolchainVersion)
 
+	args := []string{
+		toolchain,
+		"build",
+		"--bin",
+		binName,
+		"--release",
+		"--target",
+		WasmWasiTarget,
+		"--color",
+		"always",
```

Comment:
Not necessary for this PR, but if people start using this tool in CI, they may not want colors.

Reply:
Yup I thought about this too. I think we need to do a sweep on the commands in another PR to ensure we don't assume TTY, ANSI etc support.